rules/mapping.go

```diff
@@ -82,6 +82,7 @@ func newMappingRuleSnapshot(
 	), nil
 }
 
+// nolint: unparam
```

Comment:
Given that we only ever pass `false` for `tombstoned` at the moment, is it worth removing it as a parameter until we actually need to set it to something else? Since this is an internal function such a change wouldn't be public facing and we wouldn't need the special `nolint`.

Reply:
Yeah that's fair, will update